Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -759,6 +759,11 @@ def start(self):
         self.addLogObserver('stdio', self.log_observer)
         self.failedTestCount = 0
         appendCustomTestingFlags(self, self.getProperty('platform'), self.getProperty('device_model'))
+        additionalArguments = self.getProperty("additionalArguments")
+        if additionalArguments:
+            for additionalArgument in additionalArguments:
+                if additionalArgument in ["--remote-layer-tree", "--use-gpu-process"]:
+                    self.command += additionalArgument
```

Comment:
This results in each character being added individually to self.command list. e.g.: https://build.webkit-dev.org/#/builders/151/builds/3/steps/19/logs/stdio

changing `additionalArgument` to `[additionalArgument]`  works fine